plugins/core/read_detected_object_set_viame_csv.cxx

```diff
@@ -323,6 +337,56 @@ ::read_all()
       dob = std::make_shared< kwiver::vital::detected_object>( bbox, conf );
     }
 
+    if ( m_read_poly && col[i][0] == '(' ){
```

Comment:
generally you'd want to match styles of the code being modified, bracket start/end in same fashion, honoring total line spacing and not adding extra, etc; here and below